### PR TITLE
Merged two pull requests into one

### PR DESCRIPTION
**Pull request by timurkamalov**

### CODE REVIEW

File: modelimplementation/src/main/java/org/verapdf/model/ModelLoader.java [hunk 1]
```diff
@@ -25,18 +25,16 @@ private ModelLoader(){}
      * @return root object representing by {@link org.verapdf.model.coslayer.CosDict}
      * @throws FileNotFoundException
      */
-    public static CosDict getCatalog(String path) throws FileNotFoundException {
+    public static CosDict getCatalog(String path) throws IOException {
```
Comment (carlwilson) [inline modelimplementation/src/main/java/org/verapdf/model/ModelLoader.java]: Inconsistent exception documentation (IOException vs. FileNotFoundException (subclass)).

File: modelimplementation/src/main/java/org/verapdf/model/ModelLoader.java [hunk 2]
```diff
@@ -25,18 +25,16 @@ private ModelLoader(){}
...
         final File file = new File(path);
         if (!file.exists()) {
             throw new FileNotFoundException("Current file '" + path + "' not exists.");
         }
 
-        CosDict catalog = null;
+        CosDict catalog;
```
Comment (carlwilson) [inline modelimplementation/src/main/java/org/verapdf/model/ModelLoader.java]: What happens if there's no catalog?

File: modelimplementation/src/main/java/org/verapdf/model/impl/pb/cos/PBCosObject.java [hunk 3]
```diff
@@ -60,19 +42,20 @@ protected void setType(String type) {
...
-        else if (object instanceof CosObject){
-            COSBase base = ((PBCosObject) object).baseObject;
-            return  this.baseObject.equals(base);
-        } else {
-            return false;
+    public String getID() {
+        return id;
+    }
+
+    public static CosObject getFromValue(COSBase base) {
```
Comment (carlwilson) [inline modelimplementation/src/main/java/org/verapdf/model/impl/pb/cos/PBCosObject.java]: Public method commenting.

File: validationlogic/src/main/java/org/verapdf/validation/report/model/Check.java [hunk 4]
```diff
@@ -8,14 +8,14 @@
...
-    private String attr_status;
+    private String attrStatus;
 
     private CheckLocation location;
     private CheckError error;
     private boolean isHasError;
 
-    public Check(String attr_status, CheckLocation location, CheckError error, boolean isHasError) {
-        this.attr_status = attr_status;
+    public Check(String attrStatus, CheckLocation location, CheckError error, boolean isHasError) {
```
Comment (carlwilson) [inline validationlogic/src/main/java/org/verapdf/validation/report/model/Check.java]: JavaDoc for the public constructor.

File: validationlogic/src/main/java/org/verapdf/validation/report/model/CheckLocation.java [hunk 5]
```diff
@@ -10,19 +8,19 @@
  * @version 1.0
  */
 public class CheckLocation {
-    private String attr_level;
+    private String attrLevel;
     private String context;
 
-    public CheckLocation(String attr_level, String context) {
-        this.attr_level = attr_level;
+    public CheckLocation(String attrLevel, String context) {
```
Comment (carlwilson) [inline validationlogic/src/main/java/org/verapdf/validation/report/model/CheckLocation.java]: JavaDoc for public constructor.

File: validationlogic/src/test/java/org/verapdf/validation/logic/TestPDAnnot.java [hunk 6]
```diff
@@ -21,6 +21,10 @@ public TestPDAnnot(String subtype, String id) {
         this.id = id;
     }
 
+    public Boolean isContextDependent(){
```
Comment (carlwilson) [inline validationlogic/src/test/java/org/verapdf/validation/logic/TestPDAnnot.java]: Public method JavaDoc.

File: validationprofileparser/src/main/java/org/verapdf/validation/profile/model/ValidationProfile.java [hunk 7]
```diff
@@ -90,7 +90,7 @@ public String getHash() {
     public Rule getRuleById(String id){
```
Comment (carlwilson) [inline validationprofileparser/src/main/java/org/verapdf/validation/profile/model/ValidationProfile.java]: JavaDoc for public method.

File: modelimplementation/src/main/java/org/verapdf/model/ModelLoader.java (hunk 2 above)
Comment (ZhenyaM) [inline modelimplementation/src/main/java/org/verapdf/model/ModelLoader.java]: throws IOException with "Catalog cannot be found" message.

**Pull request merged**